grails-app/services/au/org/ala/ecodata/PermissionService.groovy

```diff
@@ -322,7 +322,7 @@ class PermissionService {
      * @param roles List of Hub roles that will be included in the criteria
      * @return Hub members one page at a time
      */
-    def getMembersForHubPerPage(String hubId, Integer offset, Integer max, String userId, List roles = [AccessLevel.admin, AccessLevel.caseManager, AccessLevel.readOnly]) {
```

Comment:
This is a change for MERIT but harmless to include as in current prod there isn't a way to add users with accessLevels other than the ones that were default.